teams/README.md

```diff
@@ -72,9 +72,11 @@ If you disagree with a valid veto, you must lobby the person casting the veto to
 
 | Actions            | Description                                                                               | Approval      | Binding Voters     | Minimum Length (days) |
 | :----------------- | :---------------------------------------------------------------------------------------- | :------------ | :----------------- | :-------------------- |
-| New Reviewer       | When a new reviewer is proposed for the team, should be only nominated by a maintainer.   | Lazy Majority | Active maintainers | 3                     |
+| New Reviewer       | When a new reviewer is proposed for the team, should be only nominated by a committer.    | Lazy Majority | Active maintainers | 3                     |
 | New Committer      | When a new committer is proposed for the team, should be only nominated by a maintainer.  | Consensus     | Active maintainers | 6                     |
 | New Maintainer     | When a new maintainer is proposed for the team, should be only nominated by a maintainer. | Consensus     | Active maintainers | 6                     |
 | Reviewer Removal   | When removal of review privileges is sought.                                              | Consensus     | Active maintainers | 6                     |
 | Committer Removal  | When removal of commit privileges is sought.                                              | Consensus     | Active maintainers | 6                     |
 | Maintainer Removal | When removal of maintain privileges is sought.                                            | Consensus     | Active maintainers | 6                     |
+
+Roles are not exchangeable between teams since each team might has its own management rules. But it is allowed for a team member to self-nominate him/herself for another team to reduce some roundtrip communication. 
```

Comment:
How about change `Roles are not exchangeable` to `Roles are generally not exchangeable`?

Reply:
Does this mean we still want some exceptions？

Reply:
If it's allowed for a team member to self-nominate him/herself for another team. Do we consider it as an exception? If it is not considered as exception, then `Roles are not exchangeable` is good.

Reply:
For `exchangeable`, I mean it's not allowed to exchange roles directly without vote. Although members are allowed to nominate themselves for other teams, the vote procedure is still a must.

Reply:
Ok, that makes sense.